spaceship/lib/spaceship/client.rb

```diff
@@ -186,6 +186,56 @@ def team_id=(team_id)
       @current_team_id = team_id
     end
 
+    # @return (String) The currently selected Team Name
+    def team_name
+      return @current_team_name if @current_team_name
+
+      if teams.count > 1
+        puts("The current user is in #{teams.count} teams. Pass a team Name or call `select_team` to choose a team. Using the first one for now.")
+      end
+      @current_team_name ||= teams[0]['contentProvider']['name']
+    end
+
+    # Set a new team Name which will be used from now on
+    def team_name=(team_name)
```

Comment:
I totes see the use case for the `team_name` but do we really need `team_name=`? Selected with a team name _should_ be possible with existing functionality in `select_team(team_id: nil, team_name: nil)`, right?

Reply:
@joshdholtz I don't think we really *need* `team_name=`. It's a nice to have since we have `team_id=`. We could definitely use `select_team()` function to set the `team_name`.

Feel free to reject this PR. 😄 

If you think about it we don't even need `team_id=` as well. If not, I can remove that code. LMK.

Reply:
@RishabhTayal We totally need `team_id=` still for backwards compatibility which is why I didn't recommend deleting that 🙃 But if we could 💣 that `team_name=` I think this PR is valid to add a `team_name` attribute/method